test/test_foreach.py

```diff
@@ -150,12 +150,44 @@ def _test_bin_op_list_alpha(self, device, dtype, foreach_op, foreach_op_, torch_
     # Unary ops
     #
     @dtypes(*[torch.float, torch.double, torch.complex64, torch.complex128])
```

Comment:
it would make sense to test against all floating types here and make sure that errors thrown are the same for slow and fast paths - that way we can track that bfloat16 support is on par.